sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringServices.java

```diff
@@ -42,9 +42,16 @@ public interface SpringServices
      */
     Mono<NameAvailability> checkNameAvailabilityAsync(String name, Region region);
 
-    /** @return all available sku. */
+    /**
+     * Gets all available sku.
+     *
+     * @return all available sku.
```

Comment:
nit, SKU

Reply:
Fixed by [commit#407f3a3](https://github.com/Azure/azure-sdk-for-java/pull/43458/commits/407f3a3f8ae86f6114a3a5ad46d06c60c18c6bb1)